mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -194,15 +207,9 @@ class LogisticRegressionSuite
     // thresholds and threshold must be consistent: values
     withClue("fit with ParamMap should throw error if threshold, thresholds do not match.") {
       intercept[IllegalArgumentException] {
-        lr2.fit(smallBinaryDataset,
-          lr2.thresholds -> Array(0.3, 0.7), lr2.threshold -> (expectedThreshold / 2.0))
-      }
-    }
-    withClue("fit with ParamMap should throw error if threshold, thresholds do not match.") {
-      intercept[IllegalArgumentException] {
-        val lr2model = lr2.fit(smallBinaryDataset,
-          lr2.thresholds -> Array(0.3, 0.7), lr2.threshold -> (expectedThreshold / 2.0))
-        lr2model.getThreshold
+        val lr3 = lr2.copy(
```

Comment:
I will revert this. (previously has a bug which cause this test case fail but now fixed)